test/docs-validation/main.go

```diff
@@ -31,6 +31,7 @@ var (
 		"plugin_dir",                  // deprecated
 		"runtimes",                    // printed as separate table
 		"manage_network_ns_lifecycle", // deprecated
+		"host_ip",                     // deprecated
```

Comment:
I think we don't need this any more since it's gone from the config.

Reply:
fixed!